queries/shorts/useSETHShortsQuery.ts

```diff
@@ -0,0 +1,75 @@
+import { useQuery, QueryConfig } from 'react-query';
+import { useRecoilValue } from 'recoil';
+
+import synthetix from 'lib/synthetix';
+import QUERY_KEYS from 'constants/queryKeys';
+import { appReadyState } from 'store/app';
+import { walletAddressState, isWalletConnectedState, networkState } from 'store/wallet';
+import { Synths } from 'constants/currency';
+import { ShortRewardsData } from './types';
+
+const useSETHShortsQuery = (options?: QueryConfig<ShortRewardsData>) => {
+	const isAppReady = useRecoilValue(appReadyState);
+	const isWalletConnected = useRecoilValue(isWalletConnectedState);
+	const walletAddress = useRecoilValue(walletAddressState);
+	const network = useRecoilValue(networkState);
+
```

Comment:
Please use `const isMainnet = useRecoilValue(isMainnetState);` and only do the query for Mainnet to avoid any console error on testnet or L2. (See any other pool query file for an example).

Reply:
thanks @clementbalestrat